### PR TITLE
LPS 135697 Fixes for error alert in Page Audit

### PR DESCRIPTION
**Pull request by beltranrengifo**
Motivation
=======
Error handling components for Page Audit feature need small fixes, most of them in the UI
[LPS-135855](https://issues.liferay.com/browse/LPS-135855)
[LPS-135697](https://issues.liferay.com/browse/LPS-135697)

Change summary:
---------------
* Fix the text spacings
* Fix the message

Steps to Verify:
----------------

1. Log as administrator
2. Configure an invalid api key for Page Speed
3. Launch a page audit

1. Log as administrator
2. Configure a valid api key for Page Speed
3. Launch a page audit in localhost

1. Log as administrator
2. Configure a valid api key for Page Speed
3. Try to launch a page audit in a private page

Screenshots (optional):
-----------------------
image
image

image
image

image

Checklist:
========
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] Any dependent changes have been merged and published in downs

### CODE REVIEW

**Comment by beltranrengifo** [comment]
ci:test:sf

**Comment by beltranrengifo** [comment]
ci:test:relevant

File: modules/apps/layout/layout-reports-web/src/main/resources/META-INF/resources/js/components/error-alert/ErrorAlert.js [hunk 1]
```diff
@@ -56,33 +56,32 @@ const ErrorAlert = () => {
...
-						className="font-weight-bold link-primary"
-						displayType="unstyled"
-						onClick={expandInfoHandler}
-					>
-						{actionTitle}
-					</ClayButton>
-				)}
-			</p>
+			)}
+			{userHasNotPrivileges && apiKeyError && (
```
Comment (claraizquierdo) [inline modules/apps/layout/layout-reports-web/src/main/resources/META-INF/resources/js/components/error-alert/ErrorAlert.js:69]: Why did you reorder this part of the code? Should this change be in another commit? 🤔

Comment (beltranrengifo) [reply modules/apps/layout/layout-reports-web/src/main/resources/META-INF/resources/js/components/error-alert/ErrorAlert.js:69]: Susana and I decided yesterday It was incongruent to change the element's display order. [See](https://www.figma.com/file/ubR4L2j9AxVfX71hXKS4NB/LPS-134760-why-Page-Audit-is-not-working-so-I-can-fix-the-causes?node-id=44%3A55632&viewport=-1061%2C-1397%2C0.5633594393730164)
All cases follow **title** + **text**  + **error btn**
But only one followed **title** + **error btn** + **text** 

I agree it could be in another commit but it is small and does not makes any significant difference, so I add it here...

**Review by claraizquierdo — APPROVED** [review]

File: modules/apps/layout/layout-reports-web/src/main/resources/META-INF/resources/js/components/error-alert/ErrorAlert.js [hunk 2]
```diff
@@ -64,18 +64,18 @@ const ErrorAlert = () => {
...
-					{title}
+					{title}{' '}
 				</span>
 				{pageCanNotBeAudited && (
-					<span className="ml-1">
+					<span>
 						{Liferay.Language.get(
 							'page-is-not-accessible-from-the-internet-or-is-not-indexed'
-						)}
+						)}{' '}
```
Comment (sandrodw3) [inline modules/apps/layout/layout-reports-web/src/main/resources/META-INF/resources/js/components/error-alert/ErrorAlert.js]: If our only option here is adding the space, maybe it would look a little bit better with string interpolation:
{'${Liferay.Language.get('you-can-set-the-api-key-from-site-settings-pages-google-pagespeed-insights')} '}.
What do you think? 😄

File: modules/apps/layout/layout-reports-web/test/js/components/error-alert/ErrorAlert.test.js [hunk 3]
```diff
@@ -206,6 +206,11 @@ describe('Error Alert', () => {
 			expect(
 				getByText('this-page-cannot-be-audited')
 			).toBeInTheDocument();
+			expect(
```
Comment (sandrodw3) [inline modules/apps/layout/layout-reports-web/test/js/components/error-alert/ErrorAlert.test.js:209]: I usually adapt tests in a different commit, but I guess it's up to you if you prefer doing it this way 😄

**Review by sandrodw3 — COMMENTED** [review]
Good job! 👏🏽 
Just left some tiny comments 🙂

File: modules/apps/layout/layout-reports-web/src/main/resources/META-INF/resources/js/components/error-alert/ErrorAlert.js [hunk 4]
```diff
@@ -64,18 +64,18 @@ const ErrorAlert = () => {
 						pageCanNotBeAudited ? 'font-weight-bold' : undefined
```
[the remark this answers is not in the document]
Comment (beltranrengifo) [reply modules/apps/layout/layout-reports-web/src/main/resources/META-INF/resources/js/components/error-alert/ErrorAlert.js]: Nice, thanks, I'll keep it in mind! This object approach is more readable. Anyway, this part has been removed for this PR

File: modules/apps/layout/layout-reports-web/src/main/resources/META-INF/resources/js/components/error-alert/ErrorAlert.js (hunk 2 above)
Comment (beltranrengifo) [reply modules/apps/layout/layout-reports-web/src/main/resources/META-INF/resources/js/components/error-alert/ErrorAlert.js]: I totally agree

File: modules/apps/layout/layout-reports-web/test/js/components/error-alert/ErrorAlert.test.js (hunk 3 above)
Comment (beltranrengifo) [reply modules/apps/layout/layout-reports-web/test/js/components/error-alert/ErrorAlert.test.js:209]: Uhmm... I add the cases here as they check for the string added in the commit, seemed related to me and the commit was small... I think twice next time, if you agree

Comment (sandrodw3) [reply modules/apps/layout/layout-reports-web/test/js/components/error-alert/ErrorAlert.test.js:209]: Sure! It's okay if you do it that way 🥰

**Comment by sandrodw3** [comment]
I think this is good to go!

**Comment by sandrodw3** [comment]
Fixing conflicts and forwarding here: https://github.com/liferay-tango/liferay-portal/pull/1451
